packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -35,11 +35,11 @@ Directory getApkDirectory(FlutterProject project) {
     ? project.android.buildDirectory
         .childDirectory('host')
         .childDirectory('outputs')
-        .childDirectory('apk')
```

Comment:
This doesn't work because the flutter.gradle code added doesn't get hit in the event of a flutter module. Is this something we could fix in gradle, or do I need to keep the old APK logic around too?

Presumably since the module project is generated by us we could just change something in it to output the same as regular projects?

@blasten

Reply:
yep. The `:flutter` project assigned to the module isn't the root project, which is what the plugin manipulates. That can be fixed though.